Automattic/jetpack
# Google Video: retire shortcode.

Differential Revision: D27408-code

This commit syncs r190859-wpcom.

#### Changes proposed in this Pull Request:

The Google Video Service is not available anymore, their embeds don't work anymmore, and redirects from Google Video to Youtube don't seem to work anymore either.
Let's consequently retire this embed for good, and add it to the list of unavailable shortcodes.

#### Testing instructions:

* Add the following to your site: `[googlevideo=http://video.google.com/googleplayer.swf?docId=-6006084025483872237]`
* Make sure you see a message that the service is not supported anymore on the frontend.

#### Proposed changelog entry for your changes:

* None
